runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -435,6 +438,55 @@ default boolean teleportSubmenus()
 		return false;
 	}
 
+	enum Teleports
+	{
+		WARRIORS_GUILD("Warrior's Guild", ComponentID.ADVENTURE_LOG_OPTIONS, 0),
+		FISHING_GUILD("Fishing Guild", ComponentID.ADVENTURE_LOG_OPTIONS, 1),
+		CRAFTING_GUILD("Crafting Guild", ComponentID.ADVENTURE_LOG_OPTIONS, 2),
+		FARMING_GUILD("Farming Guild", ComponentID.ADVENTURE_LOG_OPTIONS, 3),
+		OTTOS_GROTTO("Otto's Grotto", ComponentID.ADVENTURE_LOG_OPTIONS, 4),
+		RED_CHINS("Red Chinchompas", ComponentID.ADVENTURE_LOG_OPTIONS, 5, 1),
+		BLACK_CHINS("Black Chinchompas", ComponentID.ADVENTURE_LOG_OPTIONS, 5, 2),
+		HUNTER_GUILD("Hunter Guild", ComponentID.ADVENTURE_LOG_OPTIONS, 5, 3),
+		HOME("Home", ComponentID.ADVENTURE_LOG_OPTIONS, 6, 0),

```

Comment:
I would keep the naming consistent and change `Home` to `Tele to POH`

Reply:
I suppose it depends on which naming you go by, as the menu says "Home" which is largely what I went with 
![image](https://github.com/user-attachments/assets/d3ce3a3f-cae2-4af3-a970-df69ba59edc8)